### DISCUSSION TITLE
dperf工具是否有返回的功能？

### DISCUSSION THREAD

**Discussion by njfds** [baidu/dperf#453] [Q&A]
![image](https://github.com/baidu/dperf/assets/170612252/c4fe2935-1a19-4661-8ebd-39e28094dd44)
NAT网关上的报文处理逻辑：
1.	Bond0收到对端的vxlan报文。
2.	Ovs对vxlan报文进行解封装，接着 新建连接跟踪，然后转发到bond1发出。

作者您好，想请教问题如下：
使用了上图右边服务器dperf发送tcp报文，在server端接收到，显示只有tcprx，但是tcptx为0，tcp建立连接要回送确认报文，请问问什么这里会没有回送，是dperf工具没有此项功能吗？如果能得到解答十分感谢

**Comment by njfds** [comment]
![image](https://github.com/baidu/dperf/assets/170612252/8bfc3cf2-df8b-4983-a556-0b6d41a723d8)

**Comment by pengjianzhang** [comment]
你用dperf client直接打dperf server试试

**Reply by njfds** [reply]
这个就是试的结果，只有单向报文。

**Comment by pengjianzhang** [comment]
不要用bond试下
